portiaui/app/components/inspector-panel.js

```diff
@@ -45,6 +45,26 @@ export function getAttributeList(element) {
     return attributeList;
 }
 
+export function getDefaultAttribute(element) {
+    // TODO: Remove attributes that have been already annotated
```

Comment:
Attributes can be annotated multiple times. Would you be removing them just to make selecting the default attribute more useful?